staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -43,6 +43,7 @@ type Policy struct {
 	// the configured predicates even after one or more of them fails.
 	// When the flag is set to false, scheduler skips checking the rest
 	// of the predicates after it finds one predicate that failed.
+	// DEPRECATED
```

Comment:
note the config version this will be removed in?

Reply:
Per https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-parts-of-the-api, we can remove this in v2 of this type. Until then, this should remain supported.